src/OrbitGl/OpenGlBatcher.h

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GL_OPEN_GL_BATCHER_H_
+#define ORBIT_GL_OPEN_GL_BATCHER_H_
+
+#include <Batcher.h>
+
+#include <array>
+#include <memory>
+#include <unordered_map>
+#include <vector>
+
+#include "Containers/BlockChain.h"
+#include "PickingManager.h"
+
+namespace orbit_gl {
+
+namespace {
```

Comment:
You shouldn't have an anonymous namespace in a header. If you want to have these types internal put it into the namespace `orbit_gl_internal`.